openid-sharedsignals-framework-1_0.md

```diff
@@ -1719,14 +1725,15 @@ enabled
 paused
 
 > The Transmitter MUST NOT transmit events over the stream. The Transmitter
-  will hold any events it would have transmitted while paused, and SHOULD
-  transmit them when the stream’s status becomes "enabled". If a Transmitter
-  holds successive events that affect the same Subject Principal, then the
-  Transmitter MUST make sure that those events are transmitted in the order of
-  time that they were generated OR the Transmitter MUST send only the last events
-  that do not require the previous events affecting the same Subject Principal to
-  be processed by the Receiver, because the previous events are either cancelled
-  by the later events or the previous events are outdated.
+  SHOULD hold any events it would have transmitted while paused, and SHOULD
+  transmit them when the stream’s status becomes "enabled". The Transmitter MAY
+  drop some of those events. If a Transmitter holds successive events that affect
```

Comment:
Instead of "some of thoese events", how about: "The Transmitter MAY drop zero or more events that were held when the stream is paused"